recipes/perl-test-exception/meta.yaml

```diff
@@ -12,11 +12,11 @@ build:
 
 requirements:
   build:
-    - perl-threaded
+    - perl
```

Comment:
You need to bump the build number here.

Reply:
I updated `perl-threaded` to `perl` because linting failed with threaded; wasn't sure this needed a new build number. But I've updated it below; good point.